db/migrate/20151025194751_create_quotes.rb

```diff
@@ -0,0 +1,16 @@
+class CreateQuotes < ActiveRecord::Migration
+  def change
+    create_table :quotes do |t|
+      t.references :stock, index: true, foreign_key: true, null: false
+      t.date :date, index: true, null: false
+      t.float :open
```

Comment:
If these change to integers, let's change all the names to include cents:

open_cents
close_cents
high_cents

etc.

We can also add a string "currency" field with default value "USD"